src/SilverpopConnector/SilverpopXmlConnector.php

```diff
@@ -251,12 +251,17 @@ public function getListMetaData($listId) {
 	 * @return array Returns an array of SimpleXmlElement objects, one for each list
 	 * @throws SilverpopConnectorException
 	 */
-	public function getLists() {
+	public function getLists($mainFields = array()) {
```

Comment:
This new param should be listed in the doc-block. I'm also not clear on why it's named `$mainFields` as compared to... maybe `$fields` or something? That would match the similar parameter used on other methods.